extensions-contrib/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureByteSource.java

```diff
@@ -46,11 +53,19 @@ public AzureByteSource(
 
   @Override
   public InputStream openStream() throws IOException
+  {
+    return openStream(0L);
+  }
+
+  public InputStream openStream(long offset) throws IOException
   {
     try {
-      return azureStorage.getBlobInputStream(containerName, blobPath);
+      return azureStorage.getBlobInputStream(offset, containerName, blobPath);
     }
     catch (StorageException | URISyntaxException e) {
+      log.warn("Exception when opening stream to azure resource, containerName: %s, blobPath: %s, Error: %s",
```

Comment:
As an operator - what should I do if I see this warning?

Reply:
The log in RetryingInputStream does not have information about the container or blob